lib/book_settings.py

```diff
@@ -190,7 +191,8 @@ def search_for_goodreads(self, keywords):
         if not urlsearch:
             return None
 
-        return 'https://www.goodreads.com' + urlsearch.group(1)
+        url = 'https://www.goodreads.com' + urlsearch.group(1)
```

Comment:
A comment here would be good - eg. "Return the full URL with the query parameters removed"

This is related to #45 (for the benefit of github tracking only), though that was fixed an alternative way.